src/RestWrite.js

```diff
@@ -1665,6 +1665,16 @@ RestWrite.prototype._updateResponseWithData = function (response, data) {
       this.storage.fieldsChangedByTrigger.push(key);
     }
   }
+  for (const key in response) {
+    const skipKeys = ['objectId', 'createdAt', 'updatedAt', 'username'];
+    if (skipKeys.includes(key)) {
+      continue;
+    }
+    const value = response[key];
+    if (value?.__type === 'Pointer' || data[key] === value) {
```

Comment:
As long as we have Parse 4.x in LTS, let's avoid this chained optional syntax for easier backporting.

Reply:
Doesn’t babel compile this? I thought it’s only an issue if used in tests

Reply:
Lint wouldn't like it, if we backport into code into the LTS branch. To make backports easy, we are using the lowest supported syntax across all release branches.

Reply:
I believe [optional chaining](https://github.com/parse-community/parse-server/blob/5af6e5dfaa129b1a350afcba4fb381b21c4cc35d/src/Routers/SchemasRouter.js#L39) is used in many other places across the [project](https://github.com/parse-community/parse-server/blob/626fad2e71017dcc62196c487de5f908fa43000b/src/GraphQL/loaders/parseClassMutations.js#L15). My understanding was that babel compiles optional chaining to `(value === null || value === void 0 ? void 0 : value.__type)`, which the LTS branch would be able to understand. 

But no worries I can change.

Reply:
I believe [optional chaining](https://github.com/parse-community/parse-server/blob/5af6e5dfaa129b1a350afcba4fb381b21c4cc35d/src/Routers/SchemasRouter.js#L39) is used in many other places across the [project](https://github.com/parse-community/parse-server/blob/626fad2e71017dcc62196c487de5f908fa43000b/src/GraphQL/loaders/parseClassMutations.js#L15). My understanding was that babel compiles optional chaining to `(value === null || value === void 0 ? void 0 : value.__type)`, which the LTS branch would be able to understand. 

But no worries I can change.

Reply:
You are right, Babel does compile to the /lib dir, but lint checks the /src dir. So if we backport a code syntax that is not supported by node 8 (LTS 4.x Node engine is >=8), then lint will complain. Then we have to manually rewrite code and the code between alpha and the LTS branch diverges more.

Reply:
Right, thanks for the explanation! I understand now.